karma.mc.config.js

```diff
@@ -132,6 +117,9 @@ module.exports = function(config) {
             enforce: "post",
             test: /\.jsm?$/,
             loader: "istanbul-instrumenter-loader",
+            query: {
+              esModules: true,
+            },
```

Comment:
FWIW, https://webpack.js.org/concepts/loaders/#loader-features suggests that `query` has been deprecated in favor of using `options`.  I don't think it's super important that you fix this before landing, but fixing it would make the configuration more readable.

Reply:
Updated to use `options`. 

https://github.com/webpack-contrib/istanbul-instrumenter-loader#with-babel